app/scripts/controllers/preferences-controller.ts

```diff
@@ -149,125 +159,289 @@ export type PreferencesControllerState = {
   textDirection?: string;
 };
 
-export default class PreferencesController {
-  store: ObservableStore<PreferencesControllerState>;
+/**
+ * Function to get default state of the {@link PreferencesController}.
+ */
+export const getDefaultPreferencesControllerState =
+  (): PreferencesControllerState => ({
+    selectedAddress: '',
+    useBlockie: false,
+    useNonceField: false,
+    usePhishDetect: true,
+    dismissSeedBackUpReminder: false,
+    useMultiAccountBalanceChecker: true,
+    useSafeChainsListValidation: true,
+    // set to true means the dynamic list from the API is being used
+    // set to false will be using the static list from contract-metadata
+    useTokenDetection: true,
+    useNftDetection: true,
+    use4ByteResolution: true,
+    useCurrencyRateCheck: true,
+    useRequestQueue: true,
+    openSeaEnabled: true,
+    securityAlertsEnabled: true,
+    watchEthereumAccountEnabled: false,
+    bitcoinSupportEnabled: false,
+    bitcoinTestnetSupportEnabled: false,
+    ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
+    addSnapAccountEnabled: false,
+    ///: END:ONLY_INCLUDE_IF
```

Comment:
`addSnapAccountEnable` is under a feature flag, perhaps we should mark also this as optional in the state? (See comment related to `snapsAddSnapAccountModalDismissed`)